pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -81,6 +83,32 @@ func writeVifFile(buf []byte, pid, name string) error {
 	return nil
 }
 
+func setPodInterfaceCache(iface *v1.Interface, podInterfaceName string, uid string) error {
+	link, err := Handler.LinkByName(podInterfaceName)
```

Comment:
The bridge binding has similar code (for ipv4 only) for getting the ip address from the pod interface. Please reuse the same method in both of the places (at least for ipv4 since bridge binding doesn't support ipv6).

Reply:
That change seems feasible, but out of context for this bug fix
Maybe can do a separate PR for cleanup/rewrite

Reply:
This bug fix is duplicating the code, So I think it is in the scope:) But I don't mind if the fix will be done in a separate PR.